.travis.yml

```diff
@@ -6,4 +6,10 @@ python:
   - "3.5"
   - "3.6"
   - "3.7-dev"
-script: "nosetests -v --stop"
+install:
+  - pip install -r requirements.txt
+  - pip install coverage
```

Comment:
this probably isn't necessary as `python-coveralls` has `coverage` as a dependency.